odoo/models.py

```diff
@@ -2307,8 +2307,10 @@ def _read_group_raw(self, domain, fields, groupby, offset=0, limit=None, orderby
         self._apply_ir_rules(query, 'read')
         for gb in groupby_fields:
             assert gb in self._fields, "Unknown field %r in 'groupby'" % gb
-            gb_field = self._fields[gb].base_field
-            assert gb_field.store and gb_field.column_type, "Fields in 'groupby' must be regular database-persisted fields (no function or related fields), or function fields with store=True"
+            assert self._fields[gb].base_field.groupable, (
+                "Fields in 'groupby' must be regular database-persisted fields (no function or "
+                "related fields) or function fields with store=True"
```

Comment:
This error message uses pre-8.0 terminology.  It could simply be
```suggestion
                "Fields in 'groupby' must be stored fields (simple ones or many2many)"
```